modules/alibaba-ai-installing.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="alibaba-ai-installing_{context}"]
+= High-level process for creating a cluster with the {ai-full} 
+
+The main steps of the installation process are as follows:
+
+. Create the cluster with the {ai-full} and download the generated image.
+
+. Convert the image to `QCOW2` format. For details, see the section below.
```

Comment:
a11y note: Don't use positional language. "See the following section," or "see <section_name>" are preferable.